r/R/utilities.R

```diff
@@ -0,0 +1,29 @@
+#
+# subset_ids subsets a seurat object with the cell ids
+#
+subset_ids <- function(scdata, cells_id) {
+  meta_data_subset <- scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
+  current_cells <- rownames(meta_data_subset)
+  scdata <- subset(scdata, cells = current_cells)
+  return(scdata)
+}
+
+insertSetChildThroughApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```

Comment:
`insertSetChildThroughApi` seems to be about patching the cell sets, adding a new cluster to it.

This is subjective: for me, it is not obvious from that function name, which makes the code less readable. I would rename the function to show that we are saving a new cell set - like "persistCellset" or `saveCellsetToApi`, `sendCellsetToApi` or something like that.